test/src/sdk-harness/test_artifacts/storage_vec/svec_u8/tests/harness.rs

```diff
@@ -0,0 +1,25 @@
+use fuels::{prelude::*, tx::ContractId};
```

Comment:
I assume this is where the tests are meant to be however you have them in a parent file. I don't understand your structure.

Reply:
this is how the reentrancy test is doing it